videochat-conf-react/src/services/auth-service.js

```diff
@@ -24,11 +24,11 @@ class AuthService {
         mode: 1,
       },
       conference: {
-        server: process.env.REACT_APP_CONNECTYCUBE_MULTIPARTY_SERVER_ENDPOINT,
+        server: 'wss://REPLACE_APP_JANUS_DOMAIN:8989',
       },
       endpoints: {
-        api: process.env.REACT_APP_CONNECTYCUBE_API_ENDPOINT,
```

Comment:
for a sake of simplicity, let's remove endpoints configuration code (CONFERENCE_SERVER_ENDPOINT, CHAT_ENDPOINT, API_ENDPOINT )
Shared users do not need this to change